packages/paste-website/src/components/shortcodes/PageAside.tsx

```diff
@@ -10,12 +10,10 @@ interface PageAsideProps {
       slug: string;
       title: string;
     };
-    headings?: [
-      {
-        value: string;
-        depth: number;
-      }
-    ];
+    headings?: {
```

Comment:
ℹ️ Type fix: small type fix for this interface, difference is between a literal array of items, or a type that describes the item types, but doesn't define them.